content/03.results.md

```diff
@@ -186,21 +187,22 @@ This trend may have resulted from therapy-induced mutations that produced additi
 
 To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to calculate a _TP53_ score and infer _TP53_ inactivation status.
 We compared results of this classifier to "true positive" alterations derived using high-confidence SNVs, CNVs, SVs, and fusions in _TP53_.
-Specifically, we annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], or "lost" if the given patient either had a Li Fraumeni Syndrome (LFS) predisposition diagnosis, the tumor harbored a known hotspot mutation, or the tumor contained two hits (e.g. both SNV and CNV), which would suggesting both alleles had been affected.
+Specifically, we annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], or "lost" if the given patient either had a Li Fraumeni Syndrome (LFS) predisposition diagnosis, the tumor harbored a known hotspot mutation, or the tumor contained two hits (e.g. both SNV and CNV), which would suggest both alleles had been affected.
 If the _TP53_ mutation did not reside within the DNA-binding domain or we did not detect any alteration in _TP53_, we annotate the tumor as "other," reflecting its unknown _TP53_ alteration status.
-The classifier achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig4}A**). 
-By contrast, the classifier performed poorly on poly-A samples (AUROC = 0.62; **Figure {@fig:S5}A**).
-As this classifier has previously shown strong performance on poly-A data [@doi:10.1016/j.celrep.2019.09.071], the relatively smaller number of poly-A samples (N = 58) compared to stranded samples (N = 977) may explain the difference in performance across these two libraries.
+The classifier achieved a high accuracy (AUROC = 0.85) for rRNA-depleted, stranded samples compared to randomly shuffled _TP53_ scores (**Figure {@fig:Fig4}A**).
+By contrast, the classifier performed poorly on poly-A samples
+While this classifier has previously shown strong performance on poly-A data from adult tumors [@doi:10.1016/j.celrep.2019.09.071], it did not perform as well on the poly-A samples in this cohort (AUROC = 0.62; **Figure {@fig:S5}A**).
```

Comment:
```suggestion
While this classifier has previously shown strong performance on poly-A data from both adult [@doi:10.1016/j.celrep.2018.03.076] tumors and pediatric patient-derived xenografts [@doi:10.1016/j.celrep.2019.09.071], it did not perform as well on the poly-A samples in this cohort (AUROC = 0.62; **Figure {@fig:S5}A**).
```

That ref was for the PDX paper which was pediatric, but included both here for clarity. See what you think. Side note: interestingly, we ran this in OpenPedCan on TARGET+PBTA (polyA samples) and the ROC looks great, so this really is likely to be due to sample N. I think we should keep some possibilities in this statement. When we find something unexpected, it is good to hypothesize why. Some reasons may be 1) small N 2) we do not have enough TP53 mutations in the cohort. Others?

Reply:
Agree with removing "small N" as explanation.

Low TP53 mut prevalence shouldn't matter, unless there's dataset-specific normalization going on prior to classification.

The polyA low performance observation is interesting - could it be that the classifier only works with certain RNA preps? It could also be a dataset-specific normalization here too. Did you include polyA with the rest of the samples in normalization, or were these analyzed separately?

Reply:
>Low TP53 mut prevalence shouldn't matter, unless there's dataset-specific normalization going on prior to classification.

This probably fits with below, having run the datasets separately?

>Did you include polyA with the rest of the samples in normalization, or were these analyzed separately?

They were run separately so there would be no library effect in scoring. Though, if you remember, TCGA was polyA, PPTC PDX was polyA, and they performed well (on TP53). We have also run all of TARGET (polyA) separately through OpenPedCan, which performed well.